README.md

```diff
@@ -55,3 +57,21 @@ To run the app, follow these steps:
 ```flutter run```
 
 5. The app should now be running on your device or emulator.
+
+### 3.2 Running the iOS app from XCode
+
+To run the iOS app using XCode, follow these steps:
+
+1. Open the project in Xcode by running the following command in the project's root directory:
```

Comment:
XCode "C" groß -> bitte in Zukunft immer auf einheitliche Groß- und Kleinschreibung achten!

Reply:
Xcode wird mit klein “c” geschrieben. Hab überall angepasst